crates/sui/src/client_commands.rs

```diff
@@ -1829,11 +1830,12 @@ impl Display for SuiClientCommandResult {
                 }
 
                 let mut builder = TableBuilder::default();
-                builder.set_header(vec!["gasCoinId", "gasBalance"]);
+                builder.set_header(vec!["gasCoinId", "mistBalance (MIST)", "suiBalance (SUI)"]);
```

Comment:
The first time I started using `json_to_table`, I somehow decided that the table headings should match the json keys... So I kept it like that. We can of course change, not an issue.

Reply:
Okay, I figured it was something like this, but then the addition of the " (MIST)" and " (SUI)" here kind of go counter to that.  I think it would be fine to be more aesthetic in our choice of headings for the pretty tables FWIW!

Reply:
I agree, I'll go once through all of them and fix it.